src/helpers/reviews.js

```diff
@@ -1,15 +1,23 @@
 import fetch from "node-fetch";
 
-const API_URL_REVIEWS = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+const API_URL_REVIEWS =
+  "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
 
-export async function loadMovieReviews(id) {
-  const res = await fetch(`${API_URL_REVIEWS}?filters[movie]=${id}`);
+export async function loadMovieReviews(id, query) {
+  let res = "";
+  if (query.page != undefined) {
```

Comment:
What happens if the user passes in `page=foo`? I would expect a 400, but I get a 500 (because it's not handled).